### ISSUE TITLE
display all the available details of a finding

### ISSUE THREAD

**Issue by firewave** [johnthagen/clion-cppcheck#75]
```cpp
#include <string>

static char* f() {
    return std::string().c_str();
}
```

Running Cppcheck on the command-line will show the following:

```
input.cpp:4:31: error: Returning object that will be invalid when returning. [returnDanglingLifetime]
    return std::string().c_str();
                              ^
input.cpp:4:31: note: Pointer to container is created here.
    return std::string().c_str();
                              ^
input.cpp:4:31: note: Returning object that will be invalid when returning.
    return std::string().c_str();
                              ^
input.cpp:4:5: error: Dangerous usage of c_str(). The value returned by c_str() is invalid after this call. [stlcstr]
    return std::string().c_str();
    ^
input.cpp:3:0: style: The function 'f' is never used. [unusedFunction]

^
```

But CLion only shows the first message for `returnDanglingLifetime`:

![image](https://user-images.githubusercontent.com/242857/168270742-b5278681-bc97-446e-893b-265c62a24da2.png)

For some findings this information is not sufficient to actually understand why it is being reported so we should provide these additional details in the IDE.

**Comment by firewave** [comment]
I have no suggestion on how to fix it since CLion has the same problem with clang-tidy (https://youtrack.jetbrains.com/issue/CPP-19518) and certain compiler warnings (no ticket yet / some of those are shown as separate entries which might not make sense on their own) so there's not something existing to leverage AFAIK.

I do wonder if it is possible to display additional information in the tooltip:

![image](https://user-images.githubusercontent.com/242857/168271806-2ffec682-b5ba-4367-9947-077597d023b9.png)

**Comment by firewave** [comment]
Preview:

![image](https://user-images.githubusercontent.com/242857/228908896-1a5eba2d-0690-432d-891f-d02d50fc7183.png)

The `...` indicates there's more information available via the tool-tip:

![image](https://user-images.githubusercontent.com/242857/228909064-2e4903eb-70df-4e17-b29c-3478d83610e4.png)
